cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -481,6 +481,14 @@ data:
             proxy_set_header  X-Real-IP  $remote_addr;
             proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
         }
+        location = /model/savings/clusterSizingETL {
```

Comment:
I'd remove the memory intensive routing logic, then

Reply:
Great catch, thanks; done